website/docs/security/cves/CVE-2025-29928.md

```diff
@@ -4,7 +4,7 @@
 
 ### ADDENDUM May 30, 2025
 
-As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](../../releases/2025.4#sessions-are-now-stored-in-the-database) for more information.
+As of version 2025.4, the option to store sessions in cache has been removed; sessions are now exclusively stored in the database. See our [2025.4 release notes](../../releases/2025/v2025.4.md#sessions-are-now-stored-in-the-database) for more information.
```

Comment:
Does this change the URL to releases? That URL should not change as we need these links to stay the same and be consistent

Reply:
Nvm this is only the relative path, not the slug of the page